pod/perlsyn.pod

```diff
@@ -282,6 +282,15 @@ The following compound statements may be used to control flow:
 
     PHASE BLOCK
 
+As of Perl 5.36, you can iterate over multiple values at a time by specifying
+a list of lexicals within parentheses
```

Comment:
nit: This should have a period or maybe a `:`

Reply:
Similar text used `:` so I added a colon.